exonum/src/api/public/blockchain_explorer.rs

```diff
@@ -50,34 +71,66 @@ impl ExplorerApi {
         Ok(explorer.blocks_range(count, from, skip_empty_blocks))
     }
 
-    fn get_block(&self, height: Height) -> Result<Option<BlockInfo>, ApiError> {
+    fn block(&self, height: Height) -> Option<BlockInfo> {
         let explorer = BlockchainExplorer::new(&self.blockchain);
```

Comment:
`BlockchainExplorer::new` is repeated several times. Perhaps add `fn explorer(&self) -> BlockchainExplorer` method?